content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -470,10 +475,6 @@ The following configuration variables are for features that are available for us
 `DD_TRACE_LOG_PATH`
 : Sets the path for the automatic instrumentation log file and determines the directory of all other .NET Tracer log files. Ignored if `DD_TRACE_LOG_DIRECTORY` is set.
 
-`DD_TRACE_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED`
-: Enables improved resource names for web spans when set to `true`. Uses route template information where available, adds an additional span for ASP.NET Core integrations, and enables additional tags. Added in version 1.26.0. Enabled by default in 2.0.0<br>
-**Default**: `true`
-
```

Comment:
I don't think we should remove this as it's still available, and it's still a deprecated setting

Reply:
Indeed, cut instead of copy. thx

Reply:
fixed in 8b0387a48eaedd222591c2366cb2b47ff6f87c10